administrator/components/com_content/controllers/article.json.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_content
+ *
+ * @copyright   Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+ 
+ defined('_JEXEC') or die;
+/**
+ * The article controller
+ *
+ * @package     Joomla.Administrator
+ * @subpackage  com_content
+ * @since       1.6
+ */
+ class ContentControllerArticle extends JControllerForm
+{
+	    /**
+	     * Method to generate and store share token.
+	     *
+	     * @return  boolean   True if token successfully stored, false otherwise and internal error is set.
+	     *
+	     * @since   3.7
+	     */
+	     public function shareDraft()
+	     {
+	      	      $app    = JFactory::getApplication();
+		
+	    	      if (!JSession::checkToken())
+	    	      {
+	      		        echo new JResponseJson(JText::_('JINVALID_TOKEN'));
```

Comment:
This construct is wrong because the first parameter needs to be set to null because there is no dta to be returned, second parameter is the message and the third parameter needs to be set to true or false to indicate if an error occurred